test/metabase/api/user_test.clj

```diff
@@ -208,6 +211,15 @@
                                                                    :email     "cam.eron@metabase.com"}))
        (user)])))
 
+(expect
```

Comment:
what are we testing? ❓

Reply:
That we can update login_attributes after a user has been created (currently noted as a bug). I'll tadd a comment